### PR TITLE
 Create zh_cn.json

### PR DESCRIPTION
**Pull request by 200542053**

### CODE REVIEW

File: src/main/resources/assets/materialisation/lang/zh_cn.json [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  "category.materialisation.material_preparer": "物质化部件台",
+  "category.materialisation.materialising_table": "物质化工作台",
+  "block.materialisation.materialising_table": "物质化工作台",
+  "block.materialisation.material_preparer": "物质化部件台",
+  "item.materialisation.materialised_pickaxe": "复合材料镐",
+  "item.materialisation.materialised_axe": "复合材料斧",
+  "item.materialisation.materialised_shovel": "复合材料铲",
+  "item.materialisation.materialised_sword": "复合材料剑",
+  "item.materialisation.materialised_hammer": "复合材料锤",
+  "item.materialisation.materialised_megaaxe": "复合材料Mega斧",
```
Comment (shedaniel) [inline src/main/resources/assets/materialisation/lang/zh_cn.json]: Maybe let's settle on something like `巨斧`?

File: src/main/resources/assets/materialisation/lang/zh_cn.json [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+  "text.materialisation.max_durability": "耐久度上限: %d",
+  "text.materialisation.durability": "目前耐久度: %s§f (%s§f%%)",
+  "text.materialisation.breaking_speed": "破坏速度: %.2f",
+  "text.materialisation.mining_level": "挖掘等级: %d",
+  "text.materialisation.tool_handle_durability_multiplier": "耐久度倍率: %s",
+  "text.materialisation.tool_handle_speed_multiplier": "挖掘速度倍率: %s",
+  "text.materialisation.head_part_speed": "挖掘速度: %s",
+  "text.materialisation.head_part_durability": "部件耐久度: %s",
+  "text.materialisation.head_part_damage": "攻击伤害: %s",
+  "material.materialisation.minecraft.wood": "木制",
```
Comment (shedaniel) [inline src/main/resources/assets/materialisation/lang/zh_cn.json]: Rename this to only `木`

File: src/main/resources/assets/materialisation/lang/zh_cn.json [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+  "material.materialisation.minecraft.stone": "石头",
```
Comment (shedaniel) [inline src/main/resources/assets/materialisation/lang/zh_cn.json]: Rename this to only `石`

File: src/main/resources/assets/materialisation/lang/zh_cn.json [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+  "material.materialisation.netherthings.nether": "下界",
+  "material.materialisation.netherthings.glowstone": "荧石",
+  "material.materialisation.netherthings.vibranium": "振金",
+  "config.title.materialisation": "Materialisation 配置",
+  "config.title.materialisation.install_new": "安装新材料",
+  "config.button.materialisation.install": "安装新材料...",
+  "config.button.materialisation.open_folder": "打开材料包文件夹",
+  "config.button.materialisation.refresh": "刷新材料包下载界面",
+  "config.button.materialisation.download": "下载",
+  "config.text.materialisation.loading_packs": "正在寻找材料包以下载",
```
Comment (shedaniel) [inline src/main/resources/assets/materialisation/lang/zh_cn.json]: Rename this to only `正在寻找材料包`

**Review by shedaniel — CHANGES_REQUESTED** [review]

File: src/main/resources/assets/materialisation/lang/zh_cn.json (hunk 1 above)
Comment (200542053) [reply src/main/resources/assets/materialisation/lang/zh_cn.json]: Yeah, but mega feels cooler lol

Comment (shedaniel) [reply src/main/resources/assets/materialisation/lang/zh_cn.json]: But it acts as an inconsistency :)

**Review by 200542053 — COMMENTED** [review]
Update zh_cn.json

**Comment by shedaniel** [comment]
Nice work! Thanks!

**Pull request merged**